app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -66,6 +88,125 @@ public int getZoomLevel() {
+                        int minutes = Integer.valueOf(dateToParse.substring(10, 12));
+                        final int seconds = Integer.valueOf(dateToParse.substring(12, 14));
+                        final String utRel = dateToParse.substring(14, 15);
+                        final int offsetHours = Integer.parseInt(dateToParse.substring(15, 17));
+                        final int offsetMinutes = Integer.parseInt(dateToParse.substring(18, 20));
+
+                        if (utRel.equals("+")) {
+                            hours += offsetHours;
+                            minutes += offsetMinutes;
+                        } else if (utRel.equals("-")) {
+                            hours -= offsetHours;
+                            minutes -= offsetMinutes;
+                        }
+
+                        final Calendar calendar = Calendar.getInstance();
+                        calendar.set(year, month, day, hours, minutes, seconds);
+                        final Locale locale = getResources().getConfiguration().getLocales().get(0);
+                        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy", locale);
+                        final Date calendarDate = calendar.getTime();
+                        final String date = dateFormat.format(calendarDate);
+                        dateFormat = new SimpleDateFormat("HH:mm:ss", locale);
+                        final String time = dateFormat.format(calendarDate);
+                        return String.format("%s %s", date, time);
+                    } catch (Exception e) {
+                        return getString(R.string.document_properties_invalid_date);
+                    }
+                }
+
+                @Override
+                protected Void doInBackground(Void... voids) {
+                    try {
+                        final Cursor cursor = getContentResolver().query(mUri, null, null, null, null);
+                        if (cursor == null || cursor.getCount() == 0) {
+                            return null;
+                        }
+
+                        final int fileNameIndex = cursor.getColumnIndex(OpenableColumns.DISPLAY_NAME);
+                        final int fileSizeIndex = cursor.getColumnIndex(OpenableColumns.SIZE);
+
+                        cursor.moveToFirst();
+
+                        final String propertyFileName = getString(R.string.document_properties_file_name);
+                        final String fileName = cursor.getString(fileNameIndex);
+                        final String propertyFileSize = getString(R.string.document_properties_file_size);
+                        final String fileSize = parseFileSize(Long.valueOf(cursor.getString(fileSizeIndex)));
+
+                        cursor.close();
+
+                        final JSONObject json = new JSONObject(properties);
+                        final String propertyTitle = getString(R.string.document_properties_title);
```

Comment:
It seems better to change the first parameter of `formatProperty` to take an integer, call `getString` inside there and then pass these as parameters below without an extra variable for each one.